generator/test/test_util.dart

```diff
@@ -70,6 +65,11 @@ class LinesEqualsMatcher extends Matcher {
       }
     }
 
+    if (actualLines.length > expectedLines.length) {
```

Comment:
Why not check for equality. Now this will report if actual is more than expected but not if expected is more than actual.

Reply:
That case is already handled by the for loop above. If expected has more lines than actual, we would end up in the if-clause on line 58.

Reply:
why is this resolved

Reply:
On my end it looks like:
```dart
if (actualLines.length != expectedLines.length) { ...
```

Reply:
@nirinchev I agree, but there is no harm in changing it to `!=`.. it reads a bit easier.